routing/features_road_graph.cpp

```diff
@@ -272,6 +272,11 @@ IRoadGraph::RoadInfo const & FeaturesRoadGraph::GetCachedRoadInfo(FeatureID cons
   ri.m_speedKMPH = GetSpeedKMPHFromFt(ft);
   ft.SwapPoints(ri.m_points);
 
+  ft.ParseAltitude();
```

Comment:
Yes. It will be done. Using DDVector reader is a temporary code to read altitude sorted by feature id.